### PR TITLE
Fix for Issue #30 and also fixed running when project loading

### PR DESCRIPTION
**Pull request by esampsonite**

### CODE REVIEW

File: src/TSLint.MSBuild.targets [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
   <Target 
     AfterTargets="CompileTypeScript"
-    Condition="'@(TypeScriptCompile)' != ''"
+    Condition="'@(TypeScriptCompile)' != '' and '$(BuildingProject)' == 'true'"
```
Comment (JoshuaKGoldberg) [inline src/TSLint.MSBuild.targets]: Is there a reason you don't want to run the TSLint target when `BuildingProject` isn't true?

Comment (esampsonite) [inline src/TSLint.MSBuild.targets]: I'm not sure exactly *why* these particular events are triggered during project load, but I did some testing and they definitely are.  With your solution open, right click on your web project and unload it, then once it is unloaded right click and load it again.  You will see that this is executed at that time.  

The problem that I ran into was that in this particular circumstance, it kept running node over and over for ~15-20 minutes and froze Visual Studio during that time.  I have reproduced this on two different computers running VS 2015 Update 2.

Comment (JoshuaKGoldberg) [inline src/TSLint.MSBuild.targets]: Ok, thanks for the clarification. That does sound bad.

File: src/index.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ function formatOutput(lintError: string | any): string {
         return lintError;
     }
 
-    return lintError.name.replace("/", "\\")
+    return lintError.name.replace(/\//g, "\\")
```
Comment (JoshuaKGoldberg) [inline src/index.ts]: Ironically, I'm going to end up making a contrib rule for this.

**Pull request merged**